### ISSUE TITLE
Wall of Super Powers?

### ISSUE THREAD

**Issue by grahamc** [robchahin/sso-wall-of-shame#100]
I can imagine companies wanting to promote their products which don't have the SSO tax. Are you interested in these sort of contributions?

As an aside, this could create pressure on companies in ways that really matter: client base. For example, in the table a service which has a high SSO tax could link to a list of similar services which don't. Maybe this isn't the thing sso.tax wants to be, and I just want to say I have nothing invested in this or a product to sell.

**Comment by robchahin** [comment]
Intello launched something like this last year: https://www.intello.io/friends-of-sso

I don't know what hoops a vendor has to jump through to get on their list, but if it's straightforward, this seems like a good partial solution. I need to catch up on the backlog of submissions before I can opine too much on long-term changes for sso.tax

**Comment by ajhstn** [comment]
I agree, I was just about to make the same comment.  I was thinking about some of our vendors, we have some in both camps, and while it's great to point out those that Tax us, it would also be epic to praise those that don't.  The simplicity and design of this site is great and I feel should live here too somehow.

**Comment by calebsutton** [comment]
Maybe add a column to the table indicating a "Grade" ?

Where the services that increase prices by 500% or more (random example) get an "F", where more modest price changes get better grades?

**Comment by BowlesCR** [comment]
Dupe/related to #22

**Comment by JaneJeon** [comment]
@robchahin unfortunate that the page redirects to some dumb acquisition page...

**Comment by Meganitrospeed** [comment]
Correct, that page doesn't work anymore. A section of friends of SSO wouldn't hurt and would further motivate people to but use unfriendly services
